src/main/java/fi/helsinki/cs/tmc/core/commands/DownloadExercises.java

```diff
@@ -65,14 +66,15 @@ public DownloadExercises(
     }
 
     /**
-     * Constructs a new downloaded exercises command for downloading exercises of the course
-     * identified by {@code courseId} into {@code path}.
+     * Constructs a new downloaded exercises command for downloading exercises
+     * of the course identified by {@code courseId} into {@code path}.
      *
-     * @param settings      Provides login credentials and download location.
-     * @param path          Target path for downloads.
-     * @param courseId      Identifies which course's exercises should be downloaded.
-     * @param observer      This observer is notified of command's progress. May be {@code null}.
-     * @param cache         A cache for storing the downloads.
+     * @param settings Provides login credentials and download location.
+     * @param path Target path for downloads.
+     * @param courseId Identifies which course's exercises should be downloaded.
+     * @param observer This observer is notified of command's progress. May be
+     * {@code null}.
+     * @param cache A cache for storing the downloads.
```

Comment:
Nit, the @paramn <variable> should be followed by a lowercase word; no caps here please.
And omit the ending period from the at-clauses.